mozregression/fetch_configs.py

```diff
@@ -60,18 +72,23 @@ def get_build_regex(name, os, bits, processor, platprefix=r".*", platsuffix="",
     :param platsuffix: optional suffix after the platform
     :param with_ext: if True, the build extension will be appended (either
                      .zip, .tar.bz2 or .dmg depending on the os).
+    :param arch: optional arch, either x86, x86_64, aarch64. Inferred from bits if not given.
     """
+    if arch is None:
+        arch = infer_arch_from_bits(os, bits, processor)
+
     if os == "win":
-        if bits == 64:
-            if processor == "aarch64":
-                platform = r"win64-aarch64"
-            else:
-                platform = r"win64(-x86_64)?"
-            ext = r"\.zip"
+        if arch == "aarch64":
+            platform = r"win64-aarch64"
+        elif arch == "x86_64":
+            platform = r"win64(-x86_64)?"
         else:
-            platform, ext = r"win32", r"\.zip"
+            platform = r"win32"
+        ext = r"\.zip"
     elif os == "linux":
-        if bits == 64:
+        if arch == "aarch64":
+            platform, ext = r"linux-aarch64", r"\.tar.bz2"
+        elif arch == "x86_64":
             platform, ext = r"linux-x86_64", r"\.tar.bz2"
         else:
             platform, ext = r"linux-i686", r"\.tar.bz2"
```

Comment:
nit: might as well move `ext` out of the if/elif/else block since they all have the same value, to keep things more consistent. For visibility might even be good to move it to the top of the block.